setup/canary/index.md

```diff
@@ -255,9 +256,9 @@ hal config canary google disable
 
 ### Manage or view Google account information for canary
 
-You can add, delete, and multiple accounts for Google service integrations.
+You can add, edit and delete multiple accounts for Google service integrations.
```

Comment:
Nit: can you add back in the comma between edit and delete? 
(That'll be in our style guide, when we add it :) )

Reply:
As much as I'd like to defend my stance against the Oxford Comma.. 😁 change made!